supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/api/GcpBatchRequestFactory.scala

```diff
@@ -88,6 +88,7 @@ object GcpBatchRequestFactory {
                                       enableSshAccess: Boolean,
                                       //vpcNetworkAndSubnetworkProjectLabels: Option[VpcAndSubnetworkProjectLabelValues],
                                       //dockerImageCacheDiskOpt: Option[String]
+                                      gcpBatchCommand: String // TODO: Alex - verify whether we actually need this
```

Comment:
I had to propagate this command because it doesn't seem to be accessible from the existing arguments